Gordon360/Models/CCT/Housing/PreferredHall.cs

```diff
@@ -0,0 +1,27 @@
+﻿// <auto-generated> This file has been auto generated by EF Core Power Tools. </auto-generated>
+#nullable disable
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using Microsoft.EntityFrameworkCore;
+
+namespace Gordon360.Models.CCT;
+
+[PrimaryKey("ApplicationID", "Rank")]
+[Table("PreferredHall", Schema = "Housing")]
+public partial class PreferredHall
```

Comment:
I would highly recommend using the existing table structures in [CCT].[dbo] to your advantage. You are duplicating the [dbo].[Housing_HallChoices] table almost exactly here. Could you either use the existing structure or rework it to fit into your schema?

Reply:
Thank you for bringing this up! I never thought about this. The `[Housing_HallChoices]` table is the table used by Apartment Application, and the table I created here is used by Traditional Hall Application. Combining the data in these two tables together could make some things complicated🤔. Maybe we can have more discussion on this!